setup_local.sh

```diff
@@ -94,4 +149,4 @@ curl -X GET "http://${ES_HOST}:9200/shakespeare/_search?size=1"
 cd ${WDIR}
 
 echo ""
-echo "Your local environment is ready!"
+echo "Your local environment is ready."
```

Comment:
:surprised-pikachu:

Reply:
the longer I worked on this, the more it felt like that `!` was taunting me